Ginger/GingerCoreCommon/Repository/BusinessFlowLib/Activity.cs

```diff
@@ -695,7 +695,15 @@ public override void UpdateInstance(RepositoryItemBase instance, string partToUp
                     }
                     break;
                 case eItemParts.Actions:
-                    activityInstance.Acts = newInstance.Acts;
+                    if(hostItem != null)
+                    {
+                        activityInstance.Acts = newInstance.Acts;
+                        BusinessFlow currentBF = hostItem as BusinessFlow;
+                        currentBF.Activities = currentBF.Activities;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove or correct this useless self-assignment.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4582993)